.github/workflows/ci.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
       - run: npm ci
+      - run: npm run test:unit
       - run: npm run prettier:check
```

Comment:
not really part of this PR, but we should probably move the `prettier:check` one line down to make sure the code is still formatted correctly after the build.

Reply:
Moved it down 35b632c99f371df0e6fd50040aaf13b9c785a5da